guides/common/modules/con_prerequisites-for-image-based-provisioning.adoc

```diff
@@ -3,7 +3,14 @@
 [id="Prerequisites_for_Image_Based_Provisioning_{context}"]
 = Prerequisites for image-based provisioning
 
-.Post-boot configuration method
+[role="_abstract"]
+Before you can deploy pre-built cloud or virtual machine images, ensure that DNS and TFTP options are correctly configured so that hosts across multiple subnets or domains can successfully locate and retrieve the image installation content.
```

Comment:
```suggestion
Before you can deploy pre-built cloud instances or virtual machines, ensure that DNS and TFTP options are correctly configured so that hosts across multiple subnets or domains can successfully locate and retrieve the image installation content.
```

Nitpick: You use images, but you deploy/provision a cloud instance or VM.